packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -169,10 +101,7 @@ export class BlockList extends Component {
 					{ ...( Platform.OS === 'android' ? { removeClippedSubviews: false } : {} ) } // Disable clipping on Android to fix focus losing. See https://github.com/wordpress-mobile/gutenberg-mobile/pull/741#issuecomment-472746541
 					accessibilityLabel="block-list"
 					innerRef={ this.scrollViewInnerRef }
-					blockToolbarHeight={ toolbarHeight }
-					innerToolbarHeight={ blockMobileToolbarHeight }
```

Comment:
we need this to be able to show the block toolbar when we are focused on the last line of the text